pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
Hi @bitstorm ,

1.2 has been just released. 😄 

Anyway the were two reasons why to wait for this version.

1. The slash / should only be escaped after a "<" bracket (https://github.com/tdunning/open-json/commit/e1541126bf3206fa7a2b243f3791e274822128c1)
2. JSONFunction was not implemented yet. (https://github.com/tdunning/open-json/commit/174ab93ac966f0213ac4a4bd3ac5b17e0920a7c8) - this is required for json like this: { "mykey": window.myfunction() }

We can switch the version to 1.2 and it should work with this PR.

kind regards

Tobias